### PR TITLE
[1LP][RFR] Corrected checks for Projects

### PR DESCRIPTION
**Pull request by vatsalparekh**
Purpose or Intent
=================
Correction in views and class to make checking of the object for Projects and Tenants
{{ pytest: -vvvv cfme/tests/configure/test_access_control.py -k tenant }}

### CODE REVIEW

File: cfme/configure/access_control.py [hunk 1]
```diff
@@ -1326,6 +1322,8 @@ class Tenant(Updateable, BaseEntity):
     parent_tenant = attr.ib(default=None)
     _default = attr.ib(default=False)
 
+    obj_type = 'Tenant'
```
Comment (mshriver) [inline cfme/configure/access_control.py]: **Required**

Put this above L1320 please, class attributes before the object attributes.

**Comment by vatsalparekh** [comment]
59z failing due to https://bugzilla.redhat.com/show_bug.cgi?id=1511853

File: cfme/configure/access_control.py [hunk 2]
```diff
@@ -1274,7 +1271,8 @@ class DetailsTenantView(ConfigurationView):
     def is_displayed(self):
         return (
             self.accordions.accesscontrol.is_opened and
-            self.title.text == 'Tenant "{}"'.format(self.context['object'].name)
+            self.title.text == '{} "{}"'.format(self.context['object'].obj_type,
```
Comment (RonnyPfannschmidt) [inline cfme/configure/access_control.py]: @psav, @mfalesni @jkrocil  should we consider a 'self.format("{object.obj_type} "{object.name}")

File: cfme/configure/access_control.py (hunk 1 above)
Comment (vatsalparekh) [reply cfme/configure/access_control.py]: Done

**Pull request merged**